README.md

```diff
@@ -89,6 +89,31 @@ You can also manually run `nosetests` with `coverage` (but `setup.cfg` does this
 Try and get as close to 100% coverage as you can.
 
 
+## Deploy to IBM Cloud manually
+The `manifest.yml` file must be edited with the configuration of the cloud where is application is to be deployed to.
```

Comment:
do we need to locally edit this file?

Reply:
Only anyone who is using a different cloud. Since we are all sharing the same cloud instance, we don't need to edit this. And yeah, when required this needs to be changed locally.